plasmapy/physics/parameters.py

```diff
@@ -674,29 +664,34 @@ def collision_rate_ion_ion(T_i, n_i, ion_particle,
     .. [3] Callen Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf
 
     """
-    from plasmapy.physics.transport import Coulomb_logarithm
+    from plasmapy.physics.transport.collisions import Coulomb_logarithm
     T_i = T_i.to(u.K, equivalencies=u.temperature_energy())
     if coulomb_log is not None:
         coulomb_log_val = coulomb_log
     else:
         particles = [ion_particle, ion_particle]
-        coulomb_log_val = Coulomb_logarithm(T_i, n_i, particles, V)
+        coulomb_log_val = Coulomb_logarithm(T_i, n_i, particles, V, method=coulomb_log_method)
     Z_i = atomic.integer_charge(ion_particle)
     m_i = atomic.ion_mass(ion_particle)
-    nu_i = 4 / 3 * np.sqrt(np.pi / m_i) / (4 * np.pi * eps0)**2 * e**4 * \
-        n_i * Z_i**4 * coulomb_log_val / (k_B * T_i)**1.5
+    nu_i = 4 / 3 * np.sqrt(np.pi / m_i) / (4 * np.pi * eps0) ** 2 * e ** 4 * \
+           n_i * Z_i ** 4 * coulomb_log_val / (k_B * T_i) ** 1.5
     return nu_i.to(1 / u.s)
 
 
 @utils.check_quantity({
-    'n': {'units': u.m**-3, 'can_be_negative': False},
+    'n': {'units': u.m ** -3, 'can_be_negative': False},
     'T': {'units': u.K, 'can_be_negative': False},
     'B': {'units': u.T}
-})
-def Hall_parameter(n, T, B, particle, ion_particle, coulomb_log=None, V=None):
-    r"""
-    TODO
-    """
+    })
+def Hall_parameter(n,
+                   T,
+                   B,
+                   particle,
+                   ion_particle,
+                   coulomb_log=None,
+                   V=None,
+                   coulomb_log_method="classical"):
+    r"""TODO"""
```

Comment:
Open an issue and reference?

Reply:
I figured it'd be faster to write the docstring :laughing: I also changed the function's signature a bit and set `particle` to default to electrons, as that seems to be the standard case.